build.gradle

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2019 Aiven Oy
+ * Copyright 2020 Aiven Oy
```

Comment:
I'm not sure, if we should update a date. Here is a discussion, https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated#:~:text=Technically%2C%20you%20should%20update%20a,just%20to%20update%20the%20year. Most popular answer is that we shouldn't

Reply:
Well before that we upgraded. Let's keep it so far and discuss this question in the team ok?